apps/web/src/utils/updateWebTitle.ts

```diff
@@ -0,0 +1,4 @@
+
+export const updateWebTitle = (title:string) => {
```

Comment:
`updateWebTitle` is ambiguous & might be confusing. Can you change it to `setDocumentTitle`?